rover/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:8.11.4-alpine
+LABEL Unlock <ops@unlock-protocol.com>
+LABEL maintainer="ops@unlock-protocol.com"
+
+RUN npm install -g npm@6.4.1
+RUN npm install -g npm i @unlock-protocol/rover
+
+EXPOSE 8080
```

Comment:
will this conflict with locksmith, or are we going to map it to a different port in docker-compose?

Reply:
That a good call, well map it in docker-compose but ill set a default in the next release of rover